changelogs/unreleased/3167-jenting

```diff
@@ -0,0 +1 @@
+feat: support sets BackupStorageLocation CA certificate by `velero backup-location set --cacert`
```

Comment:
I would change this line to say: "feat: support setting BackupStorageLocation CA certificate via `velero backup-location set --cacert`"

Reply:
I've addressed the review comments, please take a look.